.github/workflows/openhands-resolver.yml

```
@@ -238,7 +238,7 @@ jobs:
           PYTHONPATH: ""
         run: |
           cd /tmp && python -m openhands.resolver.resolve_issue \
-            --repo ${{ github.repository }} \
+            --owner-repo ${{ github.repository }} \
```

```suggestion
            --selected_repo ${{ github.repository }} \
```

Just a thought, how about `selected_repo`?

We use it in other parts of the code, including as a command line argument, to mean the repo the user has chosen, in format "owner/repo"

---

@enyst 
Sure, I don’t have a strong opinion here. Just one thing to confirm: should it be kebab-case (selected-repo) to keep consistency with other arguments?

---

You're right! It's also used [here](https://github.com/All-Hands-AI/OpenHands/blob/51fb1fae885e2361c21027c2f38bf18ae5465d7c/openhands/core/config/utils.py#L527) for example, in headless mode.

---

updated 🫡

---

Thank you!